src/pages/_app.js

```diff
@@ -81,13 +81,15 @@ Layout.propTypes = {
 const queryCache = new QueryCache();
 const queryClient = new QueryClient({ queryCache });
 
+const isBrowser = () => typeof window !== 'undefined';
+
 // eslint-disable-next-line react/prop-types
 const App = ({ Component, pageProps, cookies }) => (
   <ContextProvider
     providers={[
       [I18nextProvider, { i18n }],
       ThemeProvider,
-      [CookiesProvider, { cookies }],
+      [CookiesProvider, { cookies: isBrowser() ? undefined : cookies }],
```

Comment:
Got this solution from https://github.com/reactivestack/cookies/issues/204#issuecomment-487365880
I tried to set `cookies` to `undefined` in `getInitialProps` if there was no `req`, so we wouldn't have to use this `isBrowser()` check, but that didn't work for some reason.
But it works, and if you want we can try to better optimize it tomorrow.

Reply:
@geoffreydhuyvetters can you take a look at this if you have some time and approve if okay or look at it together in a call? I've tried a lot of things to get it to work without a check on `window`, too many to all list here, but couldn't find any.